Makefile

```diff
@@ -125,3 +125,16 @@ android-ports: ##@other Add reverse proxy to Android Device/Simulator
 	adb reverse tcp:8081 tcp:8081
 	adb reverse tcp:3449 tcp:3449
 	adb reverse tcp:4567 tcp:4567
+
+startdev-%:
+	$(eval SYSTEM := $(word 2, $(subst -, , $@)))
+	$(eval DEVICE := $(word 3, $(subst -, , $@)))
+	${MAKE} prepare
```

Comment:
better to move `${MAKE} prepare` into `ifeq ($(SYSTEM),android)` because `prepare-ios` includes `prepare`